### PR TITLE
JC-8. done

### PR DESCRIPTION
**Pull request by aalexeen**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -77,11 +78,21 @@
                 <artifactId>liquibase-maven-plugin</artifactId>
                 <version>${liquibase-maven-plugin.version}</version>
                 <configuration>
+                    <!-- use "mvn liquibase:update -X" in Execute Maven Goal -->
                     <changeLogFile>src/main/resources/db/changelog/db.changelog-master.yaml</changeLogFile>
```
Comment (saffchen) [inline pom.xml]: Can we move this to application.yaml? Why is this included in the pom.xml?

**Review by saffchen — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (aalexeen) [reply pom.xml]: No prob. I just wanted to make it more understandable. I can remove everything))

Comment (aalexeen) [reply pom.xml]: All comments are removed.
